qubes-rpc/qubes-input-trigger-ps2

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python3
+#
+# Trigger start qubes-input-* services for dom0 ps2 input mouse/keyboard devices.
+# Used at GuiVM startup with real GPU passthrough (sys-gui-gpu)
+
+import subprocess
+import os
+import sys
+from stat import *
+
+eventFiles = os.listdir("/dev/input")
+for input_dev in eventFiles:
+    if 'event' in input_dev:  # if filename contains 'event'
+        eventFile = os.path.join("/dev/input", input_dev)
+        if S_ISCHR(os.stat(eventFile).st_mode) != 0:  # is character device?
+            udevreturn = subprocess.check_output([
+                "udevadm", "info", "--query=property", "--name=" + eventFile]).decode()
+            if 'ID_BUS=i8042' in udevreturn:
```

Comment:
This indeed works great for ps2 keyboard/mouse.

But for full laptop keyboard it seems we need yet another device, with all those extra buttons (volume up/down, brightness, etc). I don't know if there is some generic rule for that, here are few examples:
```
P: /devices/platform/topstar/input/input8/event7
N: input/event7
L: 0
S: input/by-path/platform-topstar-event
E: DEVPATH=/devices/platform/topstar/input/input8/event7
E: DEVNAME=/dev/input/event7
E: MAJOR=13
E: MINOR=71
E: SUBSYSTEM=input
E: USEC_INITIALIZED=26599847
E: ID_INPUT=1
E: ID_INPUT_KEY=1
E: ID_PATH=platform-topstar
E: ID_PATH_TAG=platform-topstar
E: LIBINPUT_DEVICE_GROUP=19/0/0:topstar
E: DEVLINKS=/dev/input/by-path/platform-topstar-event
E: TAGS=:power-switch:
```
or
```
P: /devices/platform/thinkpad_acpi/input/input8/event6
N: input/event6
S: input/by-path/platform-thinkpad_acpi-event
E: DEVLINKS=/dev/input/by-path/platform-thinkpad_acpi-event
E: DEVNAME=/dev/input/event6
E: DEVPATH=/devices/platform/thinkpad_acpi/input/input8/event6
E: ID_INPUT=1
E: ID_INPUT_KEY=1
E: ID_PATH=platform-thinkpad_acpi
E: ID_PATH_TAG=platform-thinkpad_acpi
E: KEYBOARD_KEY_01=screenlock
E: KEYBOARD_KEY_02=battery
E: KEYBOARD_KEY_03=sleep
E: KEYBOARD_KEY_04=wlan
E: KEYBOARD_KEY_06=switchvideomode
E: KEYBOARD_KEY_07=f21
E: KEYBOARD_KEY_08=f24
E: KEYBOARD_KEY_0b=suspend
E: KEYBOARD_KEY_0f=brightnessup
E: KEYBOARD_KEY_10=brightnessdown
E: KEYBOARD_KEY_11=kbdillumtoggle
E: KEYBOARD_KEY_13=zoom
E: KEYBOARD_KEY_14=volumeup
E: KEYBOARD_KEY_15=volumedown
E: KEYBOARD_KEY_16=mute
E: KEYBOARD_KEY_17=prog1
E: KEYBOARD_KEY_1a=f20
E: LIBINPUT_DEVICE_GROUP=19/17aa/5054/4101:thinkpad_acpi
E: MAJOR=13
E: MINOR=70
E: SUBSYSTEM=input
E: TAGS=:power-switch:
E: USEC_INITIALIZED=22222065
```

Also, it seems my touchpad doesn't fit here:
```
P: /devices/rmi4-00/input/input19/event14
N: input/event14
E: DEVNAME=/dev/input/event14
E: DEVPATH=/devices/rmi4-00/input/input19/event14
E: ID_BUS=rmi
E: ID_INPUT=1
E: ID_INPUT_HEIGHT_MM=52
E: ID_INPUT_TOUCHPAD=1
E: ID_INPUT_TOUCHPAD_INTEGRATION=internal
E: ID_INPUT_WIDTH_MM=96
E: LIBINPUT_DEVICE_GROUP=1d/6cb/0/0:rmi4-00
E: MAJOR=13
E: MINOR=78
E: SUBSYSTEM=input
E: USEC_INITIALIZED=22843641
```

Since it seems unrealistic to enumerate every possible laptop input device, I see two options:
 - attach everything that meets the conditions below and isn't already connected via input-proxy already (meaning: not driver uinput), or
 - have some configuration file for patterns to look for (filled with the data we collect) - this would allow users to adjust for their hardware